VotingApplication/VotingApplication.Web/Views/Create/StyleStep.cshtml

```diff
@@ -32,27 +32,61 @@
             </div>
         </md-content>
 
+        <md-content layout-gt-sm="row" layout-sm="column" layout-align="space-around center" ng-if="newPoll.PollType=='Points'" ng-init="initalisePointsConfig()" layout-padding>
```

Comment:
Should `newPoll.PollType=='Points'` be `newPoll.PollType==='Points'`? And would it be more appropriate to just expose this on the controller's scope?